.editorconfig

```diff
@@ -0,0 +1,25 @@
+# EditorConfig is awesome: https://EditorConfig.org
+root = true
+
+[*]
+charset = latin1
+end_of_line = crlf
+indent_size = 4
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[**.php]
```

Comment:
It pains me to start the war, and I will keep my own opinions out of it but most of the PHP code in the codebase is indented with tabs

Now indeed the PHP community and the PSR-2/PSR-12 prefer 4 spaces. As we are putting this in to help ensure consistency, we need it to be tabs for now.

Reply:
Oops, you're absolutely right, my bad. I _did_ check the legacy files, but I guess I got distracted and wrote the wrong value.